integration-tests/google-pubsub/src/main/java/org/apache/camel/quarkus/component/google/pubsub/it/GooglePubsubResource.java

```diff
@@ -64,7 +80,88 @@ public Response sendPojoToTopic(String fruitName) {
     @Produces(MediaType.APPLICATION_JSON)
     public Response consumePojoFromTopic() {
         Object response = consumerTemplate
-                .receiveBody("google-pubsub:{{project.id}}:{{subscription.name}}?synchronousPull=true", 5000L);
+                .receiveBody("google-pubsub:{{project.id}}:{{google-pubsub.subscription-name}}?synchronousPull=false", 5000L);
         return Response.ok(response).build();
     }
+
+    @Path("/sendToEndpoint")
+    @POST
+    public Response sentToEndpoint(String message,
+            @QueryParam("toEndpoint") String toEndpoint)
+            throws Exception {
+        producerTemplate.sendBody(toEndpoint, message);
+        return Response.created(URI.create("https://camel.apache.org")).build();
+    }
+
+    @Path("/getFromEndpoint")
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String getSchedulerResult(@QueryParam("fromEndpoint") String fromEndpoint) throws Exception {
```

Comment:
Maybe `getFromEndpoint` is a better method name ?

Reply:
renamed